Chime/Parser.m

```diff
@@ -109,22 +119,57 @@ - (void)handleVariableLine:(NSString *)line {
                  forKey:variableLabel];
 }
 
+BOOL isChimeString(NSString *string) {
+  return isStringStartingWith(string, @"\"");
+}
+
+BOOL checkBothStringEnds(NSString *string) {
+  return ('\"' == [string characterAtIndex:0]) &&
+         ('\"' == [string characterAtIndex:(string.length - 1)]);
+}
```

Comment:
Small observation here: when we get around to adding escape codes, we can check the end of the string by insisting that the last `"` character be preceded by an even number of `\`.